### PR TITLE
allow user to upgrade service manifest without upgrading code or config

### PR DESCRIPTION
**Pull request by BinWuMSFT**
this PR is to fix the issue "if the user only upgrade ServiceManifest.xml without upgrading code or config, ServiceManifest.xml won't be able to be copied to diff package"

### CODE REVIEW

File: Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1 [hunk 1]
```diff
@@ -105,8 +106,12 @@ function Create-DiffPackage
             Copy-DiffPackage -clusterPackages $clusterServiceManifest.ConfigPackage -localPackages $localServiceManifest.ConfigPackage -localParentPkgPath $localServicePkgPath -diffParentPkgPath $diffServicePkgPath
             Copy-DiffPackage -clusterPackages $clusterServiceManifest.DataPackage -localPackages $localServiceManifest.DataPackage -localParentPkgPath $localServicePkgPath -diffParentPkgPath $diffServicePkgPath
 
-            Write-Host (Get-VstsLocString -Key DIFFPKG_CopyingToDiffPackge -ArgumentList @($localServiceManifestPath, (Join-Path $diffServicePkgPath $serviceManifestName)))
-            Copy-Item -LiteralPath $localServiceManifestPath (Join-Path $diffServicePkgPath $serviceManifestName) -Force
+            Write-Host (Get-VstsLocString -Key DIFFPKG_CopyingToDiffPackge -ArgumentList @($localServiceManifestPath, $diffPkgServiceManifestPath))
+            if (!(Test-Path -PathType Container -LiteralPath $diffServicePkgPath))
+            {
+                $diffServicePkgPath = New-Item -ItemType Directory -Path $diffServicePkgPath -Force
```
Comment (bishal-pdMSFT) [inline Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1]: Why not keep it simple and always create destination dir first before copying diff packages?

File: Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1 [hunk 2]
```diff
@@ -105,8 +106,12 @@ function Create-DiffPackage
...
+            }
+            Copy-Item -LiteralPath $localServiceManifestPath $diffPkgServiceManifestPath -Force
         }
 
         Return $diffPackagePath
```
Comment (bishal-pdMSFT) [inline Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1]: You will need to bump up task patch version

Comment (bishal-pdMSFT) [inline Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1]: Is there any issue filed for this?

File: Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1 (hunk 1 above)
Comment (BinWuMSFT) [reply Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1]: @bishal-pdMSFT If a service is not changed and not copied into the diff package at all, why do we create an empty folder for the service?

File: Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1 (hunk 2 above)
Comment (BinWuMSFT) [reply Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1]: @bishal-pdMSFT There is an email thread. Forwarded to you.

File: Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1 (hunk 1 above)
Comment (bishal-pdMSFT) [reply Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1]: But we have already checked that service manifest version is greater than in cluster. line 98. So, over here you will always need to create folder and copy manifest file. Am I missing something?

Comment (BinWuMSFT) [reply Tasks/ServiceFabricDeployV1/Create-DiffPackage.psm1]: @bishal-pdMSFT You are right. We can create folder after the check

**Pull request merged**